packages/gestalt/src/Badge.css

```diff
@@ -1,16 +1,14 @@
 .Badge {
   composes: antialiased sansSerif from "./Typography.css";
-  composes: fontWeightBold from "./Typography.css";
-  composes: blue from "./Colors.css";
   composes: inlineBlock from "./Layout.css";
-  border-radius: 4px;
-  border-style: solid;
-  border-width: 1px;
+  composes: rounding1 from "./Borders.css";
+  composes: white from "./Colors.css";
+  composes: fontWeightBold from './Typography.css';
   font-size: 10px;
-  line-height: 1;
   margin-top: -4px;
+  min-height: 12px;
   padding: 2px;
-  text-transform: uppercase;
+  text-align: center;
```

Comment:
@denamwangi Can we remove this line? Since the context always expands - I don't think this will have any effect.